interface/main/left_nav.php

```diff
@@ -1053,7 +1053,22 @@ function selpopup(selobj) {
 <form method='post' name='find_patient' target='RTop'
  action='<?php echo $rootdir ?>/main/finder/patient_select.php'>
 
-<?php if ( ( $GLOBALS['concurrent_layout'] == 2) || ($GLOBALS['concurrent_layout'] == 3) ) { ?>
+<?php if ( ( $GLOBALS['concurrent_layout'] == 2) || ($GLOBALS['concurrent_layout'] == 3) ) {
+  // mdsupport - menu showhide arrows
+  if ($_SESSION['language_direction'] == 'ltr') {
```

Comment:
looks like the rtl is a bit off since it's on the left and making the left_nav to wide. should it be to the right of default above the top toggle??
(to test this, recommend using Hebrew which is a RTL language)